Tests/Form/Type/VichImageTypeTest.php

```diff
@@ -204,6 +213,45 @@ public function testLiipImagineBundleIntegration(): void
         $this->assertEquals($vars, $view->vars);
     }
 
+    public function getLiipImagineBundleIntegrationData(): array
+    {
+        $field = 'image';
+        $object = new Product();
+
+        return [
+            [
```

Comment:
can we use keys to describe provided cases?

Reply:
Done